Dockerfile

```diff
@@ -27,10 +28,10 @@ RUN apt-get update && apt-get install -y \
 	qtcreator
 
 COPY . /root/mumble
-WORKDIR /root/mumble
+WORKDIR /root/mumble/release
 
-RUN qmake -recursive main.pro CONFIG+="no-client grpc"
-RUN make release
+RUN cmake -Dclient=OFF -Dgrpc=ON ..
```

Comment:
I would explicitly use `-DCMAKE_BUILD_TYPE=Release` here. Then the Docker file does not rely on default values for the build type.

Reply:
I added `-DCMAKE_BUILD_TYPE=Release` to CMake options.